mod/item.php

```diff
@@ -136,6 +136,16 @@ function item_post(App $a) {
 			throw new HTTPException\NotFoundException(DI::l10n()->t('Unable to locate original post.'));
 		}
 
+		// When commenting on a public post then store the post for the current user
+		// This enables interaction like starring and saving into folders
```

Comment:
Can you directly star and save into folder a public item without commenting on it first ? It looks like we could use the same technique in their respective modules and show the user item action links on public items as well.

Reply:
This is one of the steps I want to have look at next - after this PR. This would include some more work at many different places (like where the options for items are created). I'm planning to make this in separate PR, since we most likely will find other places on the way.